snuba/datasets/metrics_messages.py

```diff
@@ -61,3 +66,38 @@ def value_for_counter_message(message: Mapping[str, Any]) -> Mapping[str, Any]:
     ), f"{ILLEGAL_VALUE_IN_COUNTER} {INT_FLOAT_EXPECTED}: {value}"
 
     return {"metric_type": OutputType.COUNTER.value, "count_value": value}
+
+
+def aggregation_options_for_set_message(_: Mapping[str, Any]) -> Mapping[str, Any]:
+    return {
+        "materialization_version": 1,
+        "granularities": [
+            GRANULARITY_ONE_MINUTE,
+            GRANULARITY_ONE_HOUR,
+            GRANULARITY_ONE_DAY,
+        ],
+    }
+
+
+def aggregation_options_for_distribution_message(
+    _: Mapping[str, Any]
+) -> Mapping[str, Any]:
+    return {
+        "materialization_version": 2,
```

Comment:
I would break up the PR's so that the refactoring happens in 1 PR and changing only the materialization version happens in a separate PR

Reply:
Sounds good

Reply:
https://github.com/getsentry/snuba/pull/4509